.circleci/config.yml

```diff
@@ -25,10 +25,10 @@ workflows:
       - integration-test:
           requires:
             - build
-      - e2e-test:
-          context: COVERALLS_REPO_TOKEN
-          requires:
-            - build
+      # - e2e-test:
```

Comment:
Why commenting this ?

Reply:
cause we're no longer using that e2e test (deleted the callee e2e test), only using the testing script now, so want to just disable it. 
I didn't delete the e2e script because I figured that we might want to use it in the future.